public/scripts/services/gameStateService.js

```diff
@@ -0,0 +1,26 @@
+angular.module('scrambleApp').
+  factory('gameStateService', [function() {
+    //Service to hold the state of the app across different controllers
+    var _data = {
+      state: 'start',
+      score: 0,
+      word: ''
+    };
+
+    var extend = function(src, dest) {
+      var keys = Object.keys(dest);
+      for (var i = 0; i < keys.length; i++) {
+        src[keys[i]] = dest[keys[i]];
+      }
+      return src
+    };
+
+    return {
```

Comment:
add additional calls to the lower things such as score and word, because you want to actually expose these to the top level instead of encapsulating it in the data.

    getScore: function() { return _data.score; }
    getWord : function() { return _data.word; }